tests/Unit/Console/Command/UserCreateCommandTest.php

```diff
@@ -141,6 +141,9 @@ public function testHasReviewerOption()
         $this->assertFalse($option->isArray());
     }
 
+    /**
+     * @expectedException OpenCFP\Infrastructure\Auth\UserExistsException
```

Comment:
@chartjes 

Can we remove this?

See #732.

Reply:
The test fails because it is not catching the exception. I am wondering why the test still fails

Reply:
How about rebasing on top of `master`?

Reply:
Won't this fail in the style stage since phpcs changes that to a ```$this->expectException```